build/python/backend/requirements.txt

```diff
@@ -25,11 +26,11 @@ oletools==0.56.1
 opencv-contrib-python==4.6.0.66
 opencv-python==4.6.0.66
 openpyxl==3.0.9
-pefile==2019.4.18
+pefile
```

Comment:
@ryanohoro Do we want to pin a `pefile` version here?

Reply:
Added a pin version suggested by pip.